### PR TITLE
[nojs] Add support for nojs app version through rendertron

### LINKED ISSUE #1830: [nojs] Edits for NoJS site
Most of the site works well over rendertron. Currently working on dockerizing it so that we can deploy it to the path 

` https://site.com/nojavascript`

But there are some small changes we need to make that handles special conditions. 

If the URI contains ?nojavascript=True the following changes need to be done.

1. The 5 different tabs at https://proposals.decred.org/
" Discussion,Voting,Approved,Rejected,Abandoned " all are JS triggered events. These need to become hardlinks (href) 

2 .  Some of the URL's should get rewritten. With an extra path. 
```

https://site.com/proposals/{token} => https://site.com/nojavascript/proposals/{token} 
https://site.com/user/{UUID} => https://site.com/nojavascript/user/{UUID} 
```
screenshots indicating the locations below. 
a.
image
b.
image
c.
image

These changes are only for the root / and the /proposals/* pages.

### PR DESCRIPTION
**Pull request by thi4go**
This diff adds the necessary changes to support no-javascript renders through rendertron. 

closes #1830

### CODE REVIEW

**Comment by thi4go** [comment]
Done mate @degeri

File: public/index.html [hunk 1]
```diff
@@ -34,33 +34,11 @@
...
-          <li><a href="https://github.com/decred/">Source Code on Github</a></li>
-          <li><a href="https://dcrdata.decred.org/">Block Explorer</a></li>
-          <li><a href="https://voting.decred.org/">Voting Status</a></li>
-          <li><a href="https://decred.org/downloads/">Downloads Overview</a></li>
-          </ul>
+          JavaScript is <b>disabled.</b> It is currently required to access all the functionality of Politeia. There is a 
+          basic read-only version for disabled JavaScript clients. 
+        </div><br>
+        <div>
+          <a href="/nojavascript">No JS version</a> || <a href="/">Full version</a>
```
Comment (degeri) [inline public/index.html]:
```suggestion
          <a href="/nojavascript/">No JS version</a> || <a href="/">Full version</a>
```

Comment (degeri) [reply public/index.html]: needs that "/"

File: src/containers/Header/Header.jsx [hunk 2]
```diff
@@ -5,20 +5,23 @@ import Logo from "src/componentsv2/Logo";
...
 import NavigationDrawer from "src/componentsv2/NavigationDrawer";
+import { useConfig } from "src/containers/Config";
 import styles from "./Header.module.css";
 
 const Header = ({ noBorder }) => {
   const small = useMediaQuery("(max-width: 1000px)");
   const extrasmall = useMediaQuery("(max-width: 560px)");
   const [showMenu, setShowMenu] = useState(false);
+  const { javascriptEnabled } = useConfig();
+  const url = javascriptEnabled ? "/" : "/javascript/";
```
Comment (degeri) [inline src/containers/Header/Header.jsx]: This is wrong. Why not use `NOJS_ROUTE_PREFIX` ?

Comment (thi4go) [reply src/containers/Header/Header.jsx]: My bad

File: src/containers/Header/Header.jsx [hunk 3]
```diff
@@ -5,20 +5,24 @@ import Logo from "src/componentsv2/Logo";
...
+import { useConfig } from "src/containers/Config";
+import { NOJS_ROUTE_PREFIX } from "src/constants";
 import styles from "./Header.module.css";
 
 const Header = ({ noBorder }) => {
   const small = useMediaQuery("(max-width: 1000px)");
   const extrasmall = useMediaQuery("(max-width: 560px)");
   const [showMenu, setShowMenu] = useState(false);
+  const { javascriptEnabled } = useConfig();
+  const url = javascriptEnabled ? "/" : NOJS_ROUTE_PREFIX;
```
Comment (degeri) [inline src/containers/Header/Header.jsx]: Needs "/" at the end https://site.com/nojavascript/

**Pull request merged**